ml-agents-envs/mlagents/envs/local_env_manager.py

```diff
@@ -0,0 +1,84 @@
+from typing import Any, Dict, List
+
+from mlagents.envs.base_unity_environment import BaseUnityEnvironment
+from mlagents.envs.env_manager import EnvManager, StepInfo
+from mlagents.envs.timers import timed
+from mlagents.envs import ActionInfo, BrainParameters
+
+
+class LocalEnvManager(EnvManager):
```

Comment:
@harperj I simplified this to a single environment and moved it to `ml-agents-envs`

Reply:
👍